var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -32,6 +32,11 @@ class Subversion(AutotoolsPackage):
 
     variant('serf', default=True,  description='Serf HTTP client library')
     variant('perl', default=False, description='Build with Perl bindings')
+    variant('apxs', default=True, description='Build with APXS')
+    variant('lz4_internal', default=False,
+            description='Build with internal lz4')
+    variant('utf8proc_internal', default=False,
+            description='Build with internal utf8proc')
```

Comment:
Can we avoid these variant? Using vendored dependencies is somehow working against Spack being able to manage conflicts etc.